src/main/java/mate/jdbc/service/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+
+@Service
+public class ManufacturerServiceImpl implements ManufacturerService {
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        try {
+            return manufacturerDao.get(id).get();
+        } catch (NoSuchElementException e) {
+            throw new RuntimeException("There is no manufacturer by id: " + id, e);
+        }

```

Comment:
We can simplify this
```suggestion
            return manufacturerDao.get(id).orElseThrow(() -> new RuntimeException("There is no manufacturer by id: " + id));
```